src/components/Entities/EntitySchedules.tsx

```diff
@@ -23,7 +26,11 @@ const RenderSchedules: React.FC<{
             {launchPlans.map((launchPlan, idx) => {
                 const { schedule } = launchPlan.spec.entityMetadata;
                 const frequencyString = getScheduleFrequencyString(schedule);
-                return <li key={idx}>{frequencyString}</li>;
+                const offsetString = getScheduleOffsetString(schedule);
+                const scheduleString = offsetString
+                    ? `${frequencyString} (offset: ${offsetString})`
```

Comment:
I have some concerns about the way we are displaying this.

1. Based on the tests above, it seems like this might render as something like "Every day (offset: P1D)". Personally, that feels confusing to read because I have a plainly readable string and then have to mentally parse an offset string and add it in my head to figure out when it will actually run. Is there a way to make this work the way a plain cronstring does and render the whole thing, with offset, as a readable string?
2. The UI currently renders time-related values primarily as UTC, potentially with a local time in a muted text (see the executions table). This seems like it will render a value in an arbitrary time zone?

Reply:
I will think about how to make it human readable. The offset is a duration so it's relative and not time zone bound. Together with cron, offset could be used in the scenario for example: execute it daily but against yesterday's data partition if offset is P-1D.

Reply:
I think I don't fully understand the way this cron expression is used. Offset seems to carry a different semantic meaning than simple arithmetic on the time value. I had interpreted offset to be a UTC offset.
But it sounds like it has extra logic attached to it if it is controlling what data will be used for input. In that case, I don't know if it makes sense to apply the offset to the schedule and render the resulting value.

Either way, I think this might get confusing for users. But if the way you are displaying it here makes sense to you as a user, I'm okay with shipping it as is and thinking about ways to make it more understandable.

Reply:
Yeah I totally agree this might seem odd and require more understanding of what offset is. I'm OK not to render offset for now until we figure out a better way. Internally we are integrating flyteadmin directly with a [UI system](https://github.com/spotify/backstage), so this is not critical.